acceptance-tests/dsl/build.gradle

```diff
@@ -46,7 +46,6 @@ dependencies {
   implementation 'org.web3j:abi'
   implementation 'org.web3j:besu'
   implementation 'org.web3j:crypto'
-  implementation 'tech.pegasys.ethsigner.internal:ethsigner-core'
   implementation 'tech.pegasys.signers.internal:signing-secp256k1-api'
   implementation 'tech.pegasys.signers.internal:signing-secp256k1-impl'
```

Comment:
if `tech.pegasys.signers.internal:signing-secp256k1*` dependency is not used, we might remove them as well.

Reply:
Will check if they are used

Reply:
Removed them, they were only there to bring in ethsigner